handsontable/src/plugins/manualColumnResize/manualColumnResize.js

```diff
@@ -560,6 +566,24 @@ export class ManualColumnResize extends BasePlugin {
     }
   }
 
+  /**
+   * Callback for "contextmenu" event triggered on element showing move handle. It removes handle and guide elements.
+   */
+  onContextMenu() {
+    this.hideHandleAndGuide();
+    this.hot.rootElement.removeChild(this.handle);
+    this.hot.rootElement.removeChild(this.guide);
```

Comment:
```suggestion
```
I think that it is ok that the handler does not disappear after triggering the context menu. The origin bug is still fixed, the handler does not follow the mouse after unclicking the context menu:
![Kapture 2023-06-16 at 10 22 56](https://github.com/handsontable/handsontable/assets/571316/52dbc94f-b1ae-42b2-ae7d-c8b5ac2c9f9b)

By removing these lines we do not add new code to maintain and the fix still seems to be working. So way not do it 🤔

Reply:
As you can see, there UI element left behind. Title of the issue says that: "The resize handler should be removed when user clicks RMB".

Reply:
For better code health, I'd remove the lines. UI, in this case, works fine for me. What do you think @AMBudnik @krzysztofspilka?

Reply:
To clarify. Current PR fixes the issue by hiding/removing the handler right after the RMB is clicked. I believe it's a bad UI/UX design that some elements just disappear after mouse-clicking. 
![Kapture 2023-06-16 at 12 34 01](https://github.com/handsontable/handsontable/assets/571316/69a29c24-09fd-4954-8b4c-b3cad80aa0c2)

My proposition is not to hide/remove the handler and make it just inactive for RMB.
![Kapture 2023-06-16 at 12 36 29](https://github.com/handsontable/handsontable/assets/571316/d2fefc34-f489-44cc-af94-ec5cccb105e4)

Reply:
It's fine with me 👍

Reply:
@budnix @wszymanski @AMBudnik 
The primary role of the blue highlight over the border between the headers is to indicate (visually) the possible action for the end user. In this case, the user can use left click + drag to resize the column width manually.

That action, though, is no longer available when the context menu is opened. When it's opened, left-click just closes the menu. This makes the manual-resize indicator behind the menu completely useless, right?

For the record, Google Sheets has a similar problem, but it's more consistent in its behavior because I can still resize other columns while the menu is opened.

Handsontable's behavior after this fix will be more similar to Excel than Google Sheets.

Google Sheets
https://github.com/handsontable/handsontable/assets/8048526/3c4c93b4-4795-4535-ab3a-994555e334bb

Excel
https://github.com/handsontable/handsontable/assets/8048526/33eefbf0-afd5-48ed-8ecf-d198d5ef2f48